pom.xml

```diff
@@ -22,11 +22,13 @@
         <calculator.version>1.0</calculator.version>
         <maven-surefire-plugin.version>3.0.0-M4</maven-surefire-plugin.version>
         <calculator.systemPath>${project.basedir}/src/test/resources/libs/calculator-1.0.jar</calculator.systemPath>
-        <allure.version>2.13.3</allure.version>
+        <allure.version>2.14.0</allure.version>
+        <aspectj.version>1.9.6</aspectj.version>
         <allure-maven.version>2.10.0</allure-maven.version>
         <lombok.version>1.18.20</lombok.version>
         <cucumber-java.version>6.11.0</cucumber-java.version>
-        <aspectj.version>1.9.5</aspectj.version>
+        <jdi-light-html.version>1.2.19</jdi-light-html.version>
```

Comment:
Usually the latest version of the library should be used.

Reply:
and what about this comment?

Reply:
I agree with you that need to use the latest version, but with the latest version of the aspectj errors appear and the tests do not run